ports/libmlservo/Cargo.toml

```diff
@@ -25,6 +25,7 @@ simpleservo = { path = "../libsimpleservo/api", features = ["no_static_freetype"
 rust-webvr = { version = "0.16", features = ["magicleap"] }
 webxr-api = { git = "https://github.com/servo/webxr", features = ["ipc"] }
 webxr = { git = "https://github.com/servo/webxr", features = ["ipc", "magicleap"] }
+getopts = "0.2.11"
```

Comment:
Did this belong in libsimpleservo/api/Cargo.toml instead?

Reply:
Yes, you're right. Thanks for catching!